### ISSUE TITLE
Integratiing Stack Support

### ISSUE THREAD

**Issue by chayim** [redis/go-redis#2585]
@elena-kolevska is about to start working on redis-stack support in this client, bringing along native functionality to go-redis. As part of this, @rueian and I have been discussing merging his fluent-like API here, in order to provide a "jump start" as possibilities are explored.

Let's discuss in this issue - and ensure we're all on the same page.

CC: @elena-kolevska @rueian @monkey92t

**Comment by monkey92t** [comment]
👍🏻

**Comment by rueian** [comment]
👍

**Comment by SoulPancake** [comment]
Would like to collab in some parts of this too if that's fine  ^_^ @elena-kolevska // @monkey92t

**Comment by elena-kolevska** [comment]
Of course! I'll be pushing a new branch soon and will send you a link!

**Comment by gaby** [comment]
@elena-kolevska @chayim @rueian Has there been any updates on this? I was looking into integrating Redis Stack support to [GoFiber Storage Drivers](https://github.com/gofiber/storage). We already support `go-redis` and `rueidis`.

**Comment by ndyakov** [comment]
This is more or less done. Nowadays Redis Stack is just Redis Open Source and ships with the modules included. We do support the included modules.
